std-bits/base/src/main/java/org/enso/base/cache/LRUCache.java

```diff
@@ -28,22 +28,46 @@
  * deleting entries to make space for new ones. All cache files are set to be deleted automatically
  * on JVM exit.
  *
+ * <p>Limits should be set with environment variables: ENSO_LIB_HTTP_CACHE_MAX_FILE_SIZE_MEGS --
+ * single file size, in megs ENSO_LIB_HTTP_CACHE_MAX_TOTAL_CACHE_LIMIT -- total cache size, in megs
+ * or percentage of free disk space
+ *
+ * <p>Examples: ENSO_LIB_HTTP_CACHE_MAX_FILE_SIZE_MEGS=20
+ * ENSO_LIB_HTTP_CACHE_MAX_TOTAL_CACHE_LIMIT=200 ENSO_LIB_HTTP_CACHE_MAX_TOTAL_CACHE_LIMIT=50%
+ *
  * @param <M> Additional metadata to associate with the data.
  */
 public class LRUCache<M> {
   private static final Logger logger = Logger.getLogger(LRUCache.class.getName());
 
-  private final long maxFileSize;
-  private final long maxTotalCacheSize;
-
-  private final CacheTestParameters cacheTestParameters = new CacheTestParameters();
+  /**
+   * An upper limit on the total cache size. If the cache size limit specified by the other
+   * parameters goes over this value, then this value is used.
+   */
+  private static final long MAX_TOTAL_CACHE_SIZE_FREE_SPACE_UPPER_BOUND = 100L * 1024 * 1024 * 1024;
```

Comment:
I don't understand the `FREE_SPACE` part of this name

Reply:
Removed.